recipe/meta.yaml

```diff
@@ -1,42 +1,43 @@
-{% set version = "0.11.5" %}
+{% set version = "0.12.0" %}
 
 package:
   name: lief
-  version: {{ version }}
+  version: {{ version.split('-')[0] }}
 
 source:
   - url: https://github.com/lief-project/LIEF/archive/{{ version }}.tar.gz
     fn: lief-{{ version }}.tar.gz
-    sha256: 6d6d57304a56850958e4ce54f3da2ea2b9eb856ccbab61c6cde9cba15d7c9da5
-    patches:
-      - 0001-python-Only-pass-NODEFAULTLIB-MSVCRT-when-not-BUILD_.patch
-      - 0002-Call-the-C-DLL-libLIEF.dll-so-it-does-not-clash-with.patch
-      - 0003-Add-LIEFOptions-for-BUILD_STATIC_LIBS-and-BUILD_SHAR.patch
-      - 0004-Avoid-deprecated-get_config_var-SO-use-EXT_SUFFIX-lo.patch
-      - 0005-Add-boost-throw_exception-implementation.patch
+    sha256: 4a5b893f514b03092885ef4dc0760ac8dd61860bee991e5af7f31bb33f12fc13
+    # patches:
+      # - 0001-python-Only-pass-NODEFAULTLIB-MSVCRT-when-not-BUILD_.patch
+      # - 0002-Call-the-C-DLL-libLIEF.dll-so-it-does-not-clash-with.patch
```

Comment:
It seems that the lief.dll name doesn't seem to be a problem since the tests are passing?
It might be working since it's called LIEF.dll and lief.pyd and Windows is case-sensitive?

Reply:
The names were like that when this patch was introduced. Can you try expanding the tests to do something instead of just importing?